src/platforms/f3/platform.c

```diff
@@ -34,6 +35,11 @@
 
 extern uint32_t _ebss; // NOLINT(bugprone-reserved-identifier,cert-dcl37-c,cert-dcl51-cpp)
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites